mlflow/tracking/fluent.py

```diff
@@ -481,6 +542,22 @@ def start_run(
             run_name=run_name,
         )
 
+        # If SGC run resumption is enabled, set the experiment tag mapping
+        # SGC job_run_id to this run_id for future run resumption
+        if sgc_job_run_id_tag_key:
+            try:
+                client.set_experiment_tag(
+                    exp_id_for_run, sgc_job_run_id_tag_key, active_run_obj.info.run_id
+                )
```

Comment:
On Databricks, maximum allowed tags per experiment is limited 1000. Could this be an issue?

<img width="956" height="101" alt="image" src="https://github.com/user-attachments/assets/4a084a5f-c1d4-4a88-8fab-51f8c2a22b9e" />